README.md

```diff
@@ -76,12 +76,85 @@ test a config change locally, you will need to run their open source
 [scraper](https://github.com/algolia/docsearch-scraper) against your own
 scraper to test out config changes.
 
-The Enterprise documentation uses paid algolia indices, which auto-update every
-24 hours via a [github action here](/.github/workflows/algolia.yml)
+The rest of the documentation uses paid Algolia indices, which auto-update every
```

Comment:
This is a bit perplexing. Might it make more sense to edit at lines 69ff as I suggest? Otherwise "rest of" kind of lacks context ... although I certainly understand about removing "Enterprise"